searchbar/src/styles.css

```diff
@@ -0,0 +1,93 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: grid;
+  grid-template-columns: repeat(20, 1fr);
+  grid-template-rows: repeat(3, 50px);
+  background-color: #0046be;
+  align-items: center;
+}
+
+.logo {
+  grid-column: 2 / 4;
+  grid-row: 1 / 3;
+}
+
+#logo {
+  height: 72px;
+}
+
+.middleNav {
+  grid-column: 4 / 12;
+  grid-row: 2 / 3;
+}
+
+.MuiInputBase-root {
```

Comment:
i'm flagging this mostly for me, because I think this might be what I'm using in that dropdown thats messing up.  Ill get back to you

Reply:
That probably is throwing things off! I bet there's another way for me to make the search bar white. I'll look into that too!